process/transaction/shardProcess.go

```diff
@@ -293,6 +297,7 @@ func (txProc *txProcessor) createReceiptWithReturnedGas(
 	acntSnd state.UserAccountHandler,
 	cost *big.Int,
 ) error {
+	// TODO: check why no receiptWithReturnedGas if not in sender shard
```

Comment:
because it is a move balance and all cost is taken out exactly at the sender shard.

Reply:
ok